lldb/source/Target/Statistics.cpp

```diff
@@ -353,6 +355,30 @@ llvm::json::Value DebuggerStats::ReportStatistics(
         for (const auto &symbol_module : symbol_modules.Modules())
           module_stat.symfile_modules.push_back((intptr_t)symbol_module.get());
       }
+      // Count DWO files from this symbol file using GetSeparateDebugInfo
```

Comment:
Could we put this logic into a separate function like and call it here?

Something like
```
updateDwoCounts(sym_file, total_dwo_file_count, totoal_loaded_dwo_file_count)
```